lib/game/client/game-engine.js

```diff
@@ -169,6 +192,40 @@ class GameEngine extends Game
         this.eventsManager.emit('reldens.gameEngineTabTarget', this, closerTarget, previousTarget);
     }
 
+    getScene(scene)
+    {
+        return this.engineDriver.getScene(scene);
+    }
+
+    addScene(key, sceneConfig, autoStart)
+    {
+        return this.engineDriver.addScene(key, sceneConfig, autoStart);
+    }
+
+    startScene(scene)
+    {
+        this.engineDriver.startScene(scene);
+    }
+
+    stopScene(scene)
+    {
+        this.engineDriver.stopScene(scene);
+    }
+
+    getDriver()
+    {
+        if (!this.engineDriver)
```

Comment:
engine driver should be check when this class is created, if after that is not defined then we need to throw an error once, which will prevent the rest of the execution

Reply:
Done!